src/kleinanzeigen_bot/utils/description.py

```diff
@@ -0,0 +1,70 @@
+"""
+SPDX-FileCopyrightText: © Jens Bergmann and contributors
```

Comment:
my intention for the utils package was that it only contains generic code that is not "business" related and could in theory be reused in other apps.

Reply:
Maybe this can be added to the ads.py module one level above